packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/organization/general-information/OrganizationGeneralInformation.spec.tsx

```diff
@@ -117,4 +100,66 @@ describe('Organization General Information Page', () => {
     await checkModalVisibility({ container, isVisible: true });
     await checkModalError({ container, error: 'Organization update error' });
   });
+
+  it('resets the password of the organization', async () => {
+    const { container } = await renderTest({
+      initialRoute: `/${organizationList[1].id}`,
+    });
+
+    await checkTextVisibility(
+      labels.dashboard.managed_vcd_dashboard_password_renew,
+    );
+
+    let resetPasswordLink;
+    await waitFor(
+      () => {
+        resetPasswordLink = screen.getByText(
+          labels.dashboard.managed_vcd_dashboard_password_renew,
+        );
+        return expect(resetPasswordLink).toBeEnabled();
+      },
+      { timeout: DEFAULT_TIMEOUT },
+    );
+    await waitFor(() => userEvents.click(resetPasswordLink));
+
+    await checkModalVisibility({ container, isVisible: true });
+
+    let validateButton;
+    await waitFor(() => {
+      validateButton = screen.getByText(
+        labels.dashboard.managed_vcd_dashboard_edit_modal_cta_validate,
+      );
+      return expect(validateButton).toBeEnabled();
+    });
+    await waitFor(() => userEvents.click(validateButton));
```

Comment:
I it's not I mistake of me : 

the second waitFor is not use on good use case;

- We wait for text is show : `labels.dashboard.managed_vcd_dashboard_edit_modal_cta_validate`

- And after we wait just the time of click :

await act(userEvents.click(validateButton));

But It's maybe to replace act ?

Reply:
act is deprecated